org.eclipse.draw2d-feature/build.properties

```diff
@@ -11,4 +11,5 @@
 #     IBM Corporation - initial API and implementation
 ###############################################################################
 bin.includes = feature.xml,\
-               feature.properties
\ No newline at end of file
+               feature.properties
+pom.model.groupId = org.eclipse.gef.feature
```

Comment:
shouldn't this by draw2.feature

Reply:
I suppose that's down to preference...
Because e.g. the draw2d bundle also uses `org.eclipse.gef_root` as group id.
I can change it later today.

Reply:
I just wanted to be sure that we don't run into problems. with the same groupID. I think I need to learn more maven now.

Reply:
Everything is fine as long as the group id-artifact id pair is unique.

It won't cause issues if we keep it like this, because the artifact ids (i.e. the feature ids) of all features are already distinct.

So from my perspective, the only reason to give the Draw2D and Zest features their own group id is for the sake of readability. But as I said, this comes down to preference and I don't have a strong opinion on what's the "cleaner" solution here.

Reply:
I think that keeping it as part of the GEF feature as their parent ID seems to me a good idea. Will merge it later. I have a few builds running that depend on the current GEF Classic latest.